app/controllers/rentals_controller.rb

```diff
@@ -0,0 +1,60 @@
+class RentalsController < ApplicationController
+  before_action :find_customer, only: [:create, :update]
+  before_action :find_movie, only: [:create, :update]
+
+  def create
+    rental = Rental.new(rental_params)
+    rental.save
+    @customer.movies_checked_out_count += 1
+    @customer.save
+    render json: rental.as_json(only: [:id]), status: :ok
+    return
+  end
+
+  def update
+    rental = Rental.find_by(movie_id: params[:movie_id], customer_id: params[:customer_id])
+    if rental.nil?
+      render json: {
+        ok: false,
+        errors: { rental_id: "Invalid rental id" }
+      }, status: :bad_request
+      return
+    else
+      if rental.returned == false  
+        rental.returned = true
+        rental.save
+        @customer.movies_checked_out_count -= 1
```

Comment:
If other systems interacted with the database, keeping a field to track the number of items checked out would be dangerous because it could get out of sync with the rentals in the system.  Better to make checked_out_count a method which uses rentals to determine how many movies the user has checked out.

That would be great business logic method